src/main/java/seedu/multitasky/model/util/EntryBuilder.java

```diff
@@ -82,19 +84,17 @@ public EntryBuilder withEndDateAndTime(Calendar endDateAndTime) {
 
     public Entry build() throws Exception {
         if (startDateAndTime == null) {
-            // Floating task
```

Comment:
I felt like the constructor names kinda explained what was happening already, and also the comments weren't lining up with the lines they were supposed to be describing.